tests/test_data/projects/napp_project_with_incorrect_pkg_warehouse/snowflake.yml

```diff
@@ -0,0 +1,15 @@
+definition_version: 1
```

Comment:
can we use `PdfV11Factory` instead?

Reply:
Thank you! :D You'll want to use the `ProjectV11Factory`, I see you have other files for this project.